src/components/DisplayEmbed/DisplayExternal.tsx

```diff
@@ -24,19 +23,20 @@ import { Embed, ExternalEmbed, H5pEmbed } from '../../interfaces';
 import { EmbedElement } from '../SlateEditor/plugins/embed';
 import SlateResourceBox from './SlateResourceBox';
 
-interface Props extends CustomWithTranslation {
+type EmbedType = ExternalEmbed | H5pEmbed;
+
+interface Props {
   element: EmbedElement;
   editor: Editor;
-  embed: ExternalEmbed | H5pEmbed;
+  embed: EmbedType;
   onRemoveClick: (event: MouseEvent) => void;
   language: string;
   active: boolean;
   isSelectedForCopy: boolean;
+  saveEmbedUpdates: Function;
```

Comment:
Bruk heller () => void; i stedet for Function. Då får vi validering på funksjons-signaturen.